### ISSUE TITLE
[VT-114] (m)odify can't process multiple elements

### ISSUE THREAD

**Issue by tbabej** [tbabej/vit#113]
_David Patrick on 2015-01-14T05:55:30Z says:_ 

 to reproduce this bug, try hitting "m" over a task, to modify it, and try to make more than one modification. For example

<pre>
Modify: +tag due:tomorrow
</pre> 

and the description will be changed to "+tag due:tomorrow"

**Comment by tbabej** [comment]
Migrated metadata:
```
Created: 2015-01-14T05:55:30Z
Modified: 2015-03-02T04:26:34Z
```

**Comment by tbabej** [comment]
_Scott Kostyshak on 2015-01-14T17:38:03Z says:_ 

 I cannot reproduce this. Can you try to reproduce on the development version of VIT? (checkout the git branch 1.3). Once you install the development version and try to reproduce, can you also post the output from "vit --version" ?

**Comment by tbabej** [comment]
_Koen Schoute on 2015-01-28T11:13:07Z says:_ 

 I can reproduce this. The issue occurs on my system with vit 1.2 in combination with task warrior 2.4. Installed with homebrew on mac OSX 10.9.5 . Hope that helps.

edit: a temporary workaround for users is to type 't' and then add your task number and use the 'mod' operator of taskwarrior.

**Comment by tbabej** [comment]
_Scott Kostyshak on 2015-01-28T22:20:07Z says:_ 

 Thanks for the confirmation, Koen. I'm still looking for someone to test whether this happens with the development version of VIT. If you are interested in testing the latest development version, I believe you could modify the homebrew script to checkout the branch "1.3". I understand if you don't want to take the time or risk. In that case, keep an eye out for the next version of VIT, which should hopefully be out in the next couple of months.

**Comment by tbabej** [comment]
_Scott Kostyshak on 2015-03-02T04:26:34Z says:_ 

 Fixed at 1dec1d2f for 1.3.
